submit.ipynb

```diff
@@ -65,14 +66,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "rep_gnr = RepGnr(title='Replicate')\n",
+    "#dlink((empa_viewer, 'structure'), (build_slab, 'molecule'))\n",
     "structure_selector = StructureManagerWidget(\n",
     "    importers=[\n",
     "        StructureUploadWidget(title=\"From computer\"),\n",
     "        StructureBrowserWidget(title=\"AiiDA database\"),\n",
     "        CdxmlUpload2GnrWidget(title=\"CDXML to GNR\"),\n",
     "\n",
     "    ],\n",
-    "    editors = [BasicStructureEditor()],\n",
+    "    editors = [BasicStructureEditor(),rep_gnr],\n",
```

Comment:
Let's put a tab title for the basic one as well? like `BasicStructureEditor(title='Edit')` or `BasicStructureEditor(title='Main')` or similar.